### PR TITLE
Infoblox Parser bug Fix

### PR DESCRIPTION
**Pull request by vakohl**
Removed ProcessName column from Project-Away
Renamed column DnsResponseCodeName to DnsResponseCode. Removed filter condition in Analytic Rule and updated version

   Required items, please complete
   
   Change(s):
   - Parser and Analytic Rule Update

   Reason for Change(s):
   - Bug Fix

   Version Updated:
   - from 1.0.0 to 1.0.1

   Testing Completed:
   - Done

   Checked that the validations are passing and have addressed any issues that are present:
   - Done

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/Infoblox NIOS/Parser/Infoblox_dnsclient.txt [hunk 1]
```diff
@@ -30,13 +30,13 @@ let response =
         ": query: " DnsQuery: string
         " " DnsQueryClassName: string
         " " DnsQueryTypeName: string
-        " response: " DnsResponseCodeName: string
+        " response: " DnsResponseCode: string
         " " DnsFlags: string
```
Comment (devikamehra) [inline Solutions/Infoblox NIOS/Parser/Infoblox_dnsclient.txt:34]: @vakohl Let's close the parity between old and new version. If there are a few fields extra in the new version, like EventProduct or EventVendor (Normalized Schema), that might not be a concern, but no field should be missing. Example, "Log_type" which is not used anywhere, is present in other Infoblox parser branch. We should add it here too.

**Review by devikamehra — CHANGES_REQUESTED** [review]

**Comment by v-mchatla** [comment]
Hi @vakohl,
Can you please address the Devika's comments
Thanks

**Comment by v-mchatla** [comment]
Hi @vakohl,
Can you please accommodate the changes requested by Devika.
Thanks

File: Solutions/Infoblox NIOS/Parser/Infoblox_dnsclient.txt (hunk 1 above)
Comment (vakohl) [reply Solutions/Infoblox NIOS/Parser/Infoblox_dnsclient.txt:34]: Thanks Devika. Made further changes to keep schema same as old parser

**Comment by devikamehra** [comment]
@vakohl The changes are looking good. Can you please confirm if all the OOTB content is working fine with these changes?

**Comment by v-mchatla** [comment]
Hi @vakohl,
ExcessiveNXDOMAINDNSQueries query is working fine. The other Analytic Rule PotentialDHCPStarvationAttack is having some issue with server IP field, seems it's an existing issue but can you also help in fixing this as well.
Thanks

**Comment by vakohl** [comment]
> @vakohl The changes are looking good. Can you please confirm if all the OOTB content is working fine with these changes?

thanks Devika. Yes, I did test both Analytic Rule and Workbook post this change.

**Comment by vakohl** [comment]
> PotentialDHCPStarvationAttack

This issue was old, not related to the parser change we made. This issue needs to be checked separately. hi @v-mchatla, did you got a chance to check this issue I requested earlier?

**Review by devikamehra — DISMISSED** [review]

**Comment by v-mchatla** [comment]
> > PotentialDHCPStarvationAttack
> 
> This issue was old, not related to the parser change we made. This issue needs to be checked separately. hi @v-mchatla, did you got a chance to check this issue I requested earlier?

Hi Varun,
The issue is with the serverIP field we have already added that to the custom table. Let me check once again and will reach out to you if any help is needed.
thanks

**Comment by v-mchatla** [comment]
I didn't get a chance to work on it due to some other priority work assigned. I will provide update once done.
Thanks

**Comment by v-mchatla** [comment]
Hi @vakohl - I have analysed the Analytic Rule and parser. ServerIP field is added to the custom tables, but still Analytic rule is failing. Can you also take a look and analyse. Im not able to identify the fix.
Thanks

**Comment by vakohl** [comment]
> Hi @vakohl - I have analysed the Analytic Rule and parser. ServerIP field is added to the custom tables, but still Analytic rule is failing. Can you also take a look and analyse. Im not able to identify the fix. Thanks

@v-mchatla Seems 'ServerIP' field which is used in this rule doesn't exist in schema. Replaced 'ServerIP' with 'SrcIpAddr' field in the analytic rule.

**Review by rushriva — APPROVED** [review]

**Comment by v-mchatla** [comment]
Hi @rushriva,
Can we merge this changes.
Thanks

**Comment by vakohl** [comment]
> Hi @rushriva, Can we merge this changes. Thanks

@v-mchatla please re-test the content once before merging. Like you did earlier. Thanks.

**Comment by v-mchatla** [comment]
Hi @vakohl,
The parsers are working without any issue. I will raise separate PR to make these changes available in the package by repackaging it with next version.
image
image
Thanks

**Pull request merged**